### PR TITLE
Fix: task inbox bugs

### PR DESCRIPTION
**Pull request by kumar-ish**
# Description

_Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change._
- Task pinning had a type error where the right task object wasn't being selected
- User icons of the currently-selected student were not changing from the first student selected in the task inbox on the bottom right

[NEW]

- Photo spacing and text alignment in the chat box with students
- User names are now aligned left / right more conventionally to normal apps
- Placeholder text in the search bar is now centered
- Pin icon is now centered within the task inbox

Fixes # (issue)
N/A -- issues not present as it's on a non-deployed branch

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Not rigorously -- only visually. Errors that were present before in console log are now fixed (for task pinning), and visually the user icons now change when users in the task inbox are changed.

## Testing Checklist:

- [x] Tested in latest Chrome
- [ ] Tested in latest Safari
- [ ] Tested in latest Firefox

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have requested a review from @macite and @jakerenzella on the Pull Request

### CODE REVIEW

File: src/app/units/states/tasks/inbox/directives/staff-task-list/staff-task-list.component.ts [hunk 1]
```diff
@@ -389,7 +389,7 @@ export class StaffTaskListComponent implements OnInit, OnChanges {
     }
   }
 
-  togglePin(task) {
-    task.pinned ? task.unpinTask(task) : task.pinTask(task);
+  togglePin() {
```
Comment (macite) [inline src/app/units/states/tasks/inbox/directives/staff-task-list/staff-task-list.component.ts]: This should be passed the task to pin - it is not always the current selected task

**Review by macite — DISMISSED** [review]

File: src/app/units/states/tasks/inbox/directives/staff-task-list/staff-task-list.component.ts (hunk 1 above)
Comment (jakerenzella) [reply src/app/units/states/tasks/inbox/directives/staff-task-list/staff-task-list.component.ts]: You're right @macite. @kumar-ish can you take a look what's on the filtered task object. If not might need to look at going through the task service or building out the Task object from the id in the filtered task

Comment (jakerenzella) [reply src/app/units/states/tasks/inbox/directives/staff-task-list/staff-task-list.component.ts]: Let me know if you need help

Comment (kumar-ish) [reply src/app/units/states/tasks/inbox/directives/staff-task-list/staff-task-list.component.ts]: Done -- sorry, I should have noticed this.

**Comment by kumar-ish** [comment]
Not sure how I should (visually) diff these changes in the absence of automated tests that display various actions, so for the moment I'll just do short gifs comparing them. 

The ones for the commits I did before will come later but for the moment I'll just put as I do them here -- this will just make it easier for reviewers to go commit-by-commit and see the differences. This system is a WIP, so let me know if there's anything that can be neater (any feedback is appreciated, even if it's just formatting the gifs better :)). 

NB: Student data is demo data. This displays the search bar working as intended and equally, but with the text being centered in the new one. Old and new respectively:

**Comment by kumar-ish** [comment]
For https://github.com/doubtfire-lms/doubtfire-web/pull/622/commits/6b3c0d2b1d5541fe36c6fbd03dfc752f43d03e18.

Old/new:

**Comment by jakerenzella** [comment]
This all looks good to me @kumar-ish - is there more coming for UI fixes? If so let's revert this to a draft PR until it's ready for merging! Great work!

**Comment by kumar-ish** [comment]
For https://github.com/doubtfire-lms/doubtfire-web/pull/622/commits/e9861992e9e5a40076973a2215bcc919c1777b5d:

Old/new:
old_task_info_bottom_left new_task_info_bottom_left

For https://github.com/doubtfire-lms/doubtfire-web/pull/622/commits/5fe7790e23aec709f8be5a444bf84ebdf77c9ce5:

Old/new:
old_bottom_left_task_name new_bottom_left_task_name

**Comment by kumar-ish** [comment]
For https://github.com/doubtfire-lms/doubtfire-web/pull/622/commits/93636c400c1850e20320a57be010347541c5137c:

Old/new: 
Screen Shot 2022-11-18 at 12 19 59 am Screen Shot 2022-11-18 at 12 19 20 am

**Comment by kumar-ish** [comment]
For https://github.com/doubtfire-lms/doubtfire-web/pull/622/commits/dd9cb2ab3bdde886c2f297d8044986427febd366:

Old/new:

Screen Shot 2022-11-18 at 10 47 21 am Screen Shot 2022-11-18 at 10 51 08 am

**Comment by kumar-ish** [comment]
For https://github.com/doubtfire-lms/doubtfire-web/pull/622/commits/aa4e81446546f1e62b08fd74ba8ac1e563e5efc8:

Old/new:

Screen Shot 2022-11-18 at 7 27 12 pm Screen Shot 2022-11-18 at 7 26 34 pm

**Comment by kumar-ish** [comment]
For https://github.com/doubtfire-lms/doubtfire-web/pull/622/commits/d030f8093d97f6ad85168df02653cd0cf1b51577. There's a bug / another alignment issue when there's a read icon on the right, hence the messages in here aren't aligned -- fix coming soon. Old/new (it's more noticeable when zoomed out, and looks worse in the presence of other messages):

image image

**Comment by kumar-ish** [comment]
For https://github.com/doubtfire-lms/doubtfire-web/pull/622/commits/0e97a03a491f4137ec301dd18a42130a845ebe37 and https://github.com/doubtfire-lms/doubtfire-web/pull/622/commits/c869458cd45334e009ae83ab3c911033bc1987b9. 

Old/new: 
image image

**Comment by kumar-ish** [comment]
For https://github.com/doubtfire-lms/doubtfire-web/pull/622/commits/1bfaa216440d880d82875f519e9b3f0f59ddcc23.

Old/new:
image image

**Comment by kumar-ish** [comment]
For https://github.com/doubtfire-lms/doubtfire-web/pull/622/commits/9807dfd1f18e1efc9f882a67ef99f1d198a03ff7 (literally unplayable before).

Old/new:
image image

Also oops, I went a little out of scope for this PR by changing the home page -- will keep it to task inbox now

**Comment by jakerenzella** [comment]
Hey @kumar-ish it would be really good to get this merged so the others can work from this base, is anything pending or can I merge and we can fix things from here? All looking great!

**Pull request merged**